expression/function_traits.go

```diff
@@ -87,11 +87,13 @@ var IllegalFunctions4GeneratedColumns = map[string]struct{}{
 	ast.Schema:           {},
 	ast.SessionUser:      {},
 	ast.Sleep:            {},
+	ast.Sysdate:          {},
 	ast.SystemUser:       {},
 	ast.User:             {},
 	ast.Values:           {},
 	ast.Encrypt:          {},
 	ast.Version:          {},
+	ast.JSONMerge:        {},
```

Comment:
Why the function is here?

Reply:
`JSONMerge` is a blocked function in expression of generated columns

Reply:
Why the `JSONMerge` is a "blocked func"?

Reply:
https://dev.mysql.com/doc/refman/5.7/en/json-modification-functions.html#function_json-merge
`json_merge` deprecated in MySQL 5.7.22

Reply:
If we want to be compatible with 5.7, should it be "blocked"?

Reply:
yes